frontend/src/components/dialogs/GDialog.vue

```diff
@@ -22,7 +22,7 @@ SPDX-License-Identifier: Apache-2.0
         <slot name="message">
           This is a generic dialog template.
         </slot>
-        <g-alert color="error" class="mt-4" :message.sync="message" :detailedMessage.sync="detailedMessage"></g-alert>
+        <g-message color="error" class="mt-4" :message.sync="message" :detailedMessage.sync="detailedMessage"></g-message>
```

Comment:
I cannot understand why message and detailedMessage are sync properties?

Reply:
as discussed offline we will not start this refactoring in this pr